### ISSUE TITLE
[FORMATTING] `"indentStyle": "tabularLeft"` behavior changed in v15.0.0

### ISSUE THREAD

**Issue by Phroneris** [sql-formatter-org/sql-formatter#690]
**Input data**

SQL (query.sql):

```sql
CREATE TABLE myTable (
  id INTEGER PRIMARY KEY AUTOINCREMENT,
  col1 TEXT,
  col2 TEXT
);
```

Config (config.json):

```json
{"indentStyle": "tabularLeft"}
```

Command (bash):

```shell
npx --yes sql-formatter@15.0.0 query.sql --config=config.json
```

**Expected Output**

Using `sql-formatter@14.0.0` results in this.

```sql
CREATE    TABLE myTable (
          id INTEGER PRIMARY KEY AUTOINCREMENT,
          col1 TEXT,
          col2 TEXT
          );
```

**Actual Output**

`sql-formatter@15.0.0` causes this.

```sql
CREATE    TABLE myTable (
id INTEGER PRIMARY KEY AUTOINCREMENT,
col1 TEXT,
col2 TEXT
);
```

As I read indentStyle.md, this doesn't appear to be an intentional change.

**Usage**

- How are you calling / using the library?
  - Node.js: v20.10.0
  - npm/npx: v10.2.3
  - Shell: bash via Git for Windows
  - OS: Windows 10
  - The demo page also returns this v15.0.0 result.
- What SQL language(s) does this apply to?
  - Just plain SQL, maybe.
- Which SQL Formatter version are you using?
  - v15.0.0

**Comment by nene** [comment]
Thanks for reporting.

This regression most likely happened because the default formatting of `CREATE TABLE` changed.

**Comment by nene** [comment]
So, I finally got down to fixing this regression.

But really I'm going to be deprecating this whole tabular-style feature. The formatter has largely been written to support one main indentation style, and these other styles have been more of a bolted-on features which have never received the same level of support than the default style.

Most style guides for other programming languages usually advise against such hanging-indent style of indentation. The SQLFluff tool has also [dropped support for hanging indents](https://docs.sqlfluff.com/en/stable/layout.html#hanging-indents) and the style that was supported by SQLFluff was IMHO a much better variant than what SQL Formatter supports.

**Comment by Phroneris** [comment]
The deprecation is kinda sad news for me, but thank you for working on it.

**Comment by nene** [comment]
The fix is now out in 15.1.2 release.
